test/e2e/network/kube_proxy.go

```diff
@@ -186,27 +187,25 @@ var _ = SIGDescribe("Network", func() {
 		ginkgo.By("Checking /proc/net/nf_conntrack for the timeout")
 		// If test flakes occur here, then this check should be performed
 		// in a loop as there may be a race with the client connecting.
-		e2essh.IssueSSHCommandWithResult(
+		result, err := e2essh.IssueSSHCommandWithResult(
 			fmt.Sprintf("sudo cat /proc/net/nf_conntrack | grep 'dport=%v'",
 				testDaemonTCPPort),
 			framework.TestContext.Provider,
 			clientNodeInfo.node)
+		framework.ExpectNoError(err)
 
 		// Timeout in seconds is available as the fifth column from
 		// /proc/net/nf_conntrack.
-		result, err := e2essh.IssueSSHCommandWithResult(
+		re := regexp.MustCompile(
 			fmt.Sprintf(
-				"sudo cat /proc/net/nf_conntrack "+
-					"| grep 'CLOSE_WAIT.*dst=%v.*dport=%v' "+
-					"| tail -n 1"+
```

Comment:
this took the last of multiple matching lines... is that expected? do we need to do the same?

Reply:
good question, I expect to find only conntrack entry per the tuple dst_address, dst_port and CLOSE_WAIT status and the regexp seems very restrictive.
I've added a condition to the match regexp so we check that there is only one entry matching

Reply:
I understand now the reason of this, it seems that sometimes there are stale entries or unrelated connections that may hit the regexp ... I've recovered tthis

Reply:
it turned out that the tail was masking the error because we are not setting pipefail, replaced by `grep -m 1`to avoid duplicates